eppo_core/src/event_ingestion/event_dispatcher.rs

```diff
@@ -13,7 +13,7 @@ const MIN_BATCH_SIZE: usize = 1;
 const MAX_BATCH_SIZE: usize = 10_000;
 
 #[derive(Debug, Clone)]
-pub(super) struct EventDispatcherConfig {
+pub struct EventDispatcherConfig {
```

Comment:
I would like to make fields private or mark the config as `#[non_exhaustive]`, so we can add more fields without bumping major version